src/connectors/answers/connectAnswers.ts

```diff
@@ -0,0 +1,258 @@
+import {
+  checkRendering,
+  createDocumentationMessageGenerator,
+  createConcurrentSafePromise,
+  addQueryID,
+  debounce,
+  addAbsolutePosition,
+  noop,
+  escapeHits,
+} from '../../lib/utils';
+import {
+  Connector,
+  Hits,
+  Hit,
+  FindAnswersOptions,
+  FindAnswersResponse,
+} from '../../types';
+
+type IndexWithAnswers = {
+  readonly findAnswers: any;
```

Comment:
```suggestion
  readonly findAnswers: ReturnType<SearchClient['initIndex']>['findAnswers'];
```

(but with some Required) should do the trick I think